client/pages/Login/Login.scss

```diff
@@ -0,0 +1,69 @@
+.login_container {
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  width: 100%;
+  min-height: 100vh;
+  background-image: url('http://res.cloudinary.com/dd3lv0o93/image/upload/v1531318285/landing_page.png');
+  background-color: rgba(0,0,0,.77);
+  background-blend-mode: multiply;
+  background-size: cover;
+
+  .login_container__section {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: center;
+    max-width: 1076px;
+    max-height: 596px;
+    width: 100%;
+    margin: auto;
+    padding: 0 30px;
+
+    p {
+      font-size: 44px;
+      font-weight: 300;
+      text-align: center;
+      color: #FFF;
+
+      span {
```

Comment:
So many layers of nesting. How come our scss lint dint catch this. You should clean this up and ensure to follow the BEM naming convention(https://github.com/AndelaOSP/Andela-Socials/wiki/BEM-Styleguide)